dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/domain/FilePersistenceStrategyTest.kt

```diff
@@ -267,21 +269,27 @@ internal abstract class FilePersistenceStrategyTest<T : Any>(
     }
 
     @Test
-    fun `read returns null when 1st batch is already sent but file still present`(
+    fun `reads null when batch already sent but the other thread is still trying to delete this`(
```

Comment:
I think you should still keep the old test, and add this one too.

Reply:
Well....that's the thing...the first test is not relevant anymore because now we clean the `lockedFiles` if the file is removed from the system. I can add a test maybe to return null if the file could not be deleted.